irohad/ordering/impl/on_demand_ordering_service_impl.cpp

```diff
@@ -115,9 +145,18 @@ void OnDemandOrderingServiceImpl::packNextProposals(
     if (it != current_proposals_.end()) {
       log_->debug("proposal found");
       if (not it->second.empty()) {
-        proposal_map_.emplace(round, emitProposal(round));
-        log_->debug("packNextProposal: data has been fetched for {}", round);
-        round_queue_.push(round);
+        log_->debug("Mutable proposal generation for round {}", round);
+        auto txs = getTransactions(transaction_limit_, it->second);
+        if (not txs.empty()) {
+          log_->debug("Number of transactions in proposal = {}", txs.size());
```

Comment:
I think, result of `getTransactions` should be logged independently of transactions amount

Reply:
But if there are no transactions we do not create a proposal, so this log line would be misleading.